contracts/schemes/FarmFactory.sol

```diff
@@ -172,13 +180,11 @@ contract FarmFactory {
 			address(_farm),
 			abi.encodeWithSelector(
 				_farm.notifyRewardAmount.selector,
-				_amount
+				_amount + oldBalance
```

Comment:
pls use SafeMath for mathematical operations

Reply:
It is add now